src/main/java/de/bonndan/nivio/input/kubernetes/K8sJsonParser.java

```diff
@@ -0,0 +1,55 @@
+package de.bonndan.nivio.input.kubernetes;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.bonndan.nivio.input.kubernetes.itemadapters.ItemAdapter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.Map;
+
+/**
+ * Parses config values for the crossReferenceLabel feature
+ */
+
+public class K8sJsonParser {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(K8sJsonParser.class);
+    private static final String FILEPATH = "src/main/resources/k8sLabelConfig.json";
+    private static K8sConfig k8sConfig = null;
+
+    private K8sJsonParser() {
+    }
+
```

Comment:
I loading the config once is sufficient:

```suggestion
static {
    config = getK8sConfig();
}
```